pebblo/app/config/config_validation.py

```diff
@@ -92,3 +92,16 @@ def validate_config(config_dict):
         for error in validation_errors:
             logger.error(error)
         sys.exit(1)
+
+
+def validate_cache_dir(cred_yaml):
```

Comment:
We should move this logic to `ReportsConfig.validate()`.

Reply:
This set of validation is needed before complete validation.